source/draw/gpu/opengl/OpenGLMonochrome.ooc

```diff
@@ -29,6 +31,8 @@ OpenGLMonochrome: class extends OpenGLPacked {
 		result
 	}
 	toRasterDefault: override func ~target (target: RasterImage) {
+		if (this size x % 4 != 0)
+			Debug error("Called toRasterDefault on an image which is not divisible by 4 bytes!")
 		packed := this context createRgba(IntVector2D new(this size x / 4, this size y))
```

Comment:
The RGBA image should include the padding of the raster image so that it has a chance to represent images who's width is 7 but stride is 8.